ci/travis_detect_changes.py

```diff
@@ -127,7 +127,10 @@ def get_affected_topics(affected_files):
         elif p in ('java', 'js'):
             affected[p] = True
             affected['integration'] = True
-        elif p in ('c_glib', 'integration', 'python', 'site', 'rust'):
+        elif p in ('c_glib'):
+            affected[p] = True
+            affected['ruby'] = True
```

Comment:
The Ruby wrapper binds to the C/Glib binaries?

Reply:
Yes.
It uses Apache Arrow C++ through Apache Arrow GLib.